guides/12_Client_Event_Ingestion.md

```diff
@@ -0,0 +1,45 @@
+# Event Ingestion
+
+We send the [Amazon Chime SDK meeting events](https://aws.github.io/amazon-chime-sdk-js/modules/meetingevents.html#meeting-events-and-attributes) to the Amazon Chime backend to analyze meeting health trends or identify common failures. This helps to improve your meeting experience.
```

Comment:
Do we need a guide for this? Seem like just a use case in the README would suffice.

Reply:
Since, we currently only internally send the events governed by MeetingHistoryState and do not support sending custom events hence did not add a use case for this.